core/baseline-core-v1.0-psd01.md

```diff
@@ -191,8 +191,273 @@ The ability of a Party operating Workflows on a baseline-compliant implementatio
+
+Requesters are only aware of the identity and commercial data of their Provider(s), but not of the other participants in the supply chain. However, Requesters can cryptographically verify that a given set of claims by Providers about the supply chain are true, for example, that all supply chain participants are not located in an embargo country.
+
+Therefore, a pseudonymous map of a supply chain is a cryptographically connected and verifiable list of proofs about the relationships of participants and integrity of supply chain events that does not disclose identifying details of Providers and their commercial data.
+
+This allows enforcement of conformance with regulations, additional legal and technical requirements without disclosure of confidential information.
+
+## 2.8 Baseline Protocol Reference Architecture 
+
+This section describes the components of the Baseline Reference Architecture 
+* (Commercial) State Synchornization
+* DLTs and BPI/DLT Abstraction Layers
+* External Applications
+* Baseline Protocol Stack Detailed Reference Architecture Layers and Componants
+
+### 2.8.1 (Commercial) State Synchornization
+
+A BPI can be used  as a common frame of reference for business processes that can be used in a complementary way to existing System-of-Record integrations.
+
+#### Illustrative High-Level Example
+
+A Master Services Agreement (MSA) between a Requester (Buyer) and a Provider (Seller) is implemented on a BPI and contains billing terms, pricing, discounts, and Seller information such as billing address etc. Once established and agreed upon by Buyer and Seller, the BPI provides state synchronization between Buyer and Seller since the ERP systems for Buyer and Seller can now refer to mutually agreed upon data as a common frame of reference. Based on this mutually agreed upon state in the MSA, the Buyer creates an Order in the business workflow based on the MSA and a cryptographic proof (in zero knowledge) that confirms not only the correct application of business logic but also correct application of commercial data in the Order creation. This proof is submitted together with the Order through the BPI and then validated by the Seller without having to utilize its own System of Record for validation using the BPI. If the proof is validated, the Seller accepts the proposed state change by generating its own cryptographic proof confirming its acceptance of the state change. The Seller then updates the state of the business workflow in the BPI and sends the new proof to the Buyer. 
+
+The figure below visually demonstrates high-level Buyer and Seller Order generation and acceptance assuming that a MSA between Buyer and Seller already exists and is recorded on a BPI, and that the commercial state has been synchronized up to this workstep in the commercial business workflow.
```

Comment:
Which steps in the figure are automated vs which ones require some sort of manual interaction (i.e. click on a button in UI to sign a document so that the proof can be stored on DLT)?

Reply:
they could be all automated and all manual or a mixture. Why is that important in this context? @bitwiseguy